message_parser/src/main/java/cumulus_message_adapter/message_parser/AdapterLogger.java

```diff
@@ -0,0 +1,174 @@
+package cumulus_message_adapter.message_parser;
+
+import com.amazonaws.services.lambda.runtime.Context; 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import com.google.gson.Gson;
+import java.util.Map;
+import java.util.Stack;
+import java.util.HashMap;
+import java.sql.Timestamp;
+import java.util.Date;
+
+/**
+ * Utilities for logging adapter messages
+ */
+public class AdapterLogger
+{
+    static final Logger _logger = LogManager.getLogger();
+
+    static final String LEVEL_ERROR = "error";
+    static final String LEVEL_FATAL = "fatal";
+    static final String LEVEL_WARNING = "warn";
+    static final String LEVEL_INFO = "info";
+    static final String LEVEL_DEBUG = "debug";
+    static final String LEVEL_TRACE = "trace";
```

Comment:
Unrelated to this PR but I think we should put this down somewhere and make sure all the three implementations support the same levels.